docs/sources/tutorials/get-started.md

```diff
@@ -129,19 +130,49 @@ Paste this component next in the `config.alloy` file:
 
 ```alloy
 loki.source.file "log_scrape" {
-    targets    = local.file_match.local_files.targets
-    forward_to = [loki.write.grafana_loki.receiver]
-    tail_from_end = true
+   targets    = local.file_match.local_files.targets
+   forward_to = [loki.process.filter_logs.receiver]
+   tail_from_end = true
 }
 ```
 
 This configuration creates a [loki.source.file][] component named `log_scrape`, and shows the pipeline concept of {{< param "PRODUCT_NAME" >}} in action. The `log_scrape` component does the following:
 
 1. It connects to the `local_files` component (its "source" or target).
-1. It forwards the logs it scrapes to the "receiver" of another component called `grafana_loki` that you will define next.
+1. It forwards the logs it scrapes to the "receiver" of another component called `filter_logs` that you will define next.
 1. It provides extra attributes and options, in this case, you will tail log files from the end and not ingest the entire past history.
 
-### Third component: Write logs to Loki
+### Third component: Filter non-essential logs 
+
+Filtering non-essential logs before shipping them to a data source can help you manage log volumes to reduce costs. The filtering strategy of each organization will differ as they have different monitoring needs and setups. 
+
+The following example is intended to demonstrate how logs can be filtered out (“dropped”) before being shipped to Loki. 
+
+Paste this component next in the `config.alloy` file:
+```alloy
+loki.process "filter_logs" {
+  stage.drop {
+       source = ""
+       expression  = ".*Connection closed by authenticating user root"
+       drop_counter_reason = "irrelevant"
```

Comment:
Maybe a different name here? The logs could be relevant, but perhaps we want to drop some because they're just too noisy :sweat_smile: 
```suggestion
       drop_counter_reason = "noisy"
```